<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hackrf transmitter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>96</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QSlider" name="txvgaSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>24</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>62</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QwtPlot" name="inputSpectrum">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="freqselector">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>select frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="copyrightLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectName">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="txvga_display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="inputRate_Label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="frequencyDisplay">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QSlider" name="scopeAmplifier">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>24</width>
      <height>160</height>
     </rect>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
